src/goto-symex/goto_symex_state.h

```diff
@@ -223,15 +223,16 @@ class goto_symex_statet final : public goto_statet
   }
 
   /// Drops an L1 name from the local L2 map
-  void drop_l1_name(symex_renaming_levelt::current_namest::const_iterator it)
+  void drop_existing_l1_name(const irep_idt &l1_identifier)
   {
-    level2.current_names.erase(it);
+    level2.current_names.erase(l1_identifier);
   }
```

Comment:
I think this method should either be removed or else be used from `drop_l1_name`.

Reply:
The other version is now using `erase_if_exists()` as suggested by @smowton